### PR TITLE
docs: update sample_data command

### PR DESCRIPTION
**Pull request by pengge**

### CODE REVIEW

File: src/zh/guide/reference/sample_data.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ https://s3.amazonaws.com/noaa.water-database/NOAA_data.txt
 ```shell
 curl -G  https://s3.amazonaws.com/noaa.water-database/NOAA_data.txt > NOAA_data.txt
 
-ts-cli -import -path=NOAA_data.txt -host=127.0.0.1 -port=8086 -precision=s
+ts-cli import --path=NOAA_data.txt --host=127.0.0.1 --port=8086 --precision=s
```
Comment (shilinlee) [inline src/zh/guide/reference/sample_data.md:19]: Thanks for your contribution. 

Please update the cmd in the English version. -> `src/guide/reference/sample_data.md`

And sign your DCO.

**Comment by shilinlee** [comment]
The DCO is **required** !

**Comment by pengge** [comment]
> The DCO is **required** !

Done

**Review by shilinlee — APPROVED** [review]
LGTM

**Pull request merged**